coba/environments/filters.py

```diff
@@ -997,6 +997,8 @@ def filter(self, interactions: Iterable[Interaction]) -> Iterable[Interaction]:
                 new['actions'] = list(map(methodcaller('copy'),new['actions']))
 
             if not action_materialized and is_dense_action:
+                #new['action'] = list(new['action'])
```

Comment:
related to this previous PR - https://github.com/VowpalWabbit/coba/pull/24

Reply:
https://github.com/VowpalWabbit/coba/actions/runs/4128674424/jobs/7133406460#step:10:131